### ISSUE TITLE
Triple clicking a block causes overselection in webkit

### ISSUE THREAD

**Issue by johnpuddephatt** [wymeditor/wymeditor#717]
As stated in the title, triple-clicking a block in Chrome/Safari causes what I can only describe as 'overselection'. This is clearly visible in the screenshot, where the gray highlighted selection extends beyond the white background of the paragraph. It appears to be standard behaviour on all webpages, not something that only occurs with WYM.

![screenshot 2015-05-05 23 00 48](https://cloud.githubusercontent.com/assets/8030910/7483867/9dc951e8-f37a-11e4-83cf-19005013e195.png)

The problem is that when this occurs the selection cannot be formatted in any way – applying bold or a heading for example does nothing.

I'm not very familiar with rangy or WYMeditor so I don't know the best way to prevent this, but adding the following to the .selectedContainer function (just after $focus is defined as a jQuery object) does seem to work:

```
if ($focus[0].nodeType != WYMeditor.NODE_TYPE.TEXT && $anchor[0].nodeType === WYMeditor.NODE_TYPE.TEXT) {
    selection.selectAllChildren($anchor[0]);
}
```

**Comment by winhamwr** [comment]
Good catch! It looks like we should definitely support styling a container after a triple-click. Any thoughts, @mightyiam ?
